src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -1285,8 +1285,8 @@ class StackChunkAllocator : public MemAllocator {
     assert(_word_size > _stack_size, "");
 
     // zero out fields (but not the stack)
-    const size_t hs = oopDesc::header_size();
-    Copy::fill_to_aligned_words(mem + hs, vmClasses::StackChunk_klass()->size_helper() - hs);
+    const size_t hs = oopDesc::base_offset_in_bytes();
+    Copy::fill_to_bytes((char*)mem + hs, vmClasses::StackChunk_klass()->size_helper() * HeapWordSize - hs);
```

Comment:
Likely performance-sensitive path. Can we do the same "adjust if only aligned by 4 bytes, then do aligned words copy", as in other places?